src/elements/progress-spinner/progress-spinner.md

```diff
@@ -0,0 +1,51 @@
+### Indeterminate
+
+Indicates infinite loading.
+
+```jsx
+<ProgressSpinner />
+```
+
+### Controlled
+
+Indicates the actual progress value.
+
+```jsx
+import { useEffect, useRef, useState } from 'react'
+import Button from '../button';
+
+const AnimatedProgress = () => {
+  const timeoutRef = useRef()
```

Comment:
```suggestion
  const timeoutRef = useRef(null)
```

Reply:
I do not really see a big difference between `null` and `undefined` in this case

Reply:
There is no difference you are right. I just prefer explicitly put param to that hook because if you have some type check it shows you a warning.